servers/cu/src/domain/event-transport.js

```diff
@@ -0,0 +1,101 @@
+import Libhoney from 'libhoney'
+import Database from 'better-sqlite3'
+import fs from 'fs'
+
+export class CompositeTransport {
+  constructor (transports) {
+    this.transports = transports // An array of transport instances
+  }
+
+  sendEvents (events, processId, nonce) {
+    this.transports.forEach((transport) => {
+      transport.sendEvents(events, processId, nonce)
+    })
+  }
+}
+
+export class ConsoleTransport {
+  sendEvents (events, processId, nonce) {
+    console.log(
+      `[ProcID: ${processId}; Nonce: ${nonce}]: Vacuumed events:`,
+      events
+    )
+  }
+}
+
+export class HoneycombTransport {
+  constructor (writeKey, dataset, apiHost = 'https://api.honeycomb.io', dbFilePath = './largest_nonces.db') {
```

Comment:
I'm not sure if it's incongruous with the rest of the code base, but I generally pass in objects to avoid depending on order when there are this many args.

Reply:
The fact that I'm using classes already makes this somewhat incongruous 😂 I may add this.

Reply:
a837df57